### ISSUE TITLE
npm ERR! code 1 bower 1.6.6

### ISSUE THREAD

**Issue by pokaxperia** [bower/bower.github.io#185]
After update bower from 1.6.2 to 1.6.6, this error appears:

![screen shot 2015-11-25 at 7 33 25 pm](https://cloud.githubusercontent.com/assets/1803166/11413146/72372a7e-93ab-11e5-9709-a6d3af4c933e.png)

![screen shot 2015-11-25 at 7 29 17 pm](https://cloud.githubusercontent.com/assets/1803166/11413105/e2529434-93aa-11e5-980d-1bfed6216c9e.png)

npm ERR! code 1

RESOLVED, sorry !!!1

**Comment by faceleg** [comment]
So this is no longer an issue?

**Comment by visikov** [comment]
What did you do to solve?

**Comment by fernandoguedes** [comment]
Please, best describes your problem and share with us the solution.

**Comment by marcoslhc** [comment]
That looked lika an outdated npm, but please @pokaxperia share with us the solution so we can close the issue and rest assure is solved

**Comment by sheerun** [comment]
I've released new bower that should fix the problem. Could you confirm?

**Comment by pokaxperia** [comment]
Let me explain:

1.- I receive a message to update bower from 1.6.2 to 1.6.6 (first image).
2.- After update bower, I receive many warnings.
3.- I tried install some bower package, then appears a few warnings and one error (second image).
4.- I tried to update npm, then update bower again, but the same error persists.
5.- Then install "sudo npm install readable-stream " and "sudo npm install minimist ".

And it works.

Greetings
